playbooks/roles/notify-failure-rocket/defaults/main.yml

```diff
@@ -0,0 +1,24 @@
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+rc_domain: "chat.suse.de"
+rc_protocol: "https"
+rc_url: "{{ rc_protocol }}://{{ rc_domain }}"
+rc_token: ""
```

Comment:
can we leave this undefined, so that the role fails when this is not defined, instead of silently not working?

Reply:
mmmh, that may hit us due to lazy_loading or instant loading. I dont remember which one does the lazy one (include_role vs import_role) but yeah we could do that.

Reply:
It's not necessary.

Reply:
woops, its now removed